content/docs/user-guide/dvc-files.md

```diff
@@ -153,9 +153,7 @@ stages: # Required
     cmd:
       - pip install -r requirements.txt
       - >- # YAML multiline string block
-        python ${train.script}
-        --out model.pkl
-        --thresh ${train.thresh}
+        python ${train.script} --out model.pkl --thresh ${train.thresh}
```

Comment:
Hmmm, I think I want to keep that YAML Block Chomp... If not in this doc, in a coming one.

Why is this invalid for Prettier? Thoughts @rogermparent ?

Reply:
I believe this is due to the `proseWrap: always` setting that we have in `.prettierrc` as opposed to the default `preserve`. I've run into issues with `always` before, particularly with custom components.

Switching to `preserve` will solve both of these issues, and `proseWrap` only affects MD files so the impact won't make unintended changes to prettier rules in actual code.

Reply:
Thanks @rogermparent I applied the change in 5cffe8cc and that automatically closed this PR. Let's see how that goes 🙂

Reply:
prettier rules are important for MD. We consider MD as part of code. What are the consequences of changing it?

Reply:
Not much. See #2060

Reply:
Since everything is already formatted this experiment only proves that it preserves most of the things. The question is how does it behave in different "broken" cases.